torch/csrc/autograd/variable.h

```diff
@@ -34,6 +34,20 @@ struct Variable : public at::Tensor {
   /*implicit*/ Variable(Tensor const & rhs) : Tensor(rhs) {}
   /*implicit*/ Variable(Tensor && rhs) noexcept : Tensor(std::move(rhs)) {}
 
+  Edge gradient_edge() const {
+    // If grad_fn is null (as is the case for a leaf node), we instead
+    // interpret the gradient function to be a grad accumulator,
+    // which will accumulate its inputs into the grad property of the
+    // variable. These nodes get suppressed in some situations,
+    // see "suppress grad accumulation" below. Note that only variables which
+    // have `requires_grad = True` can have grad accumulators.
+    if (const auto gradient = grad_fn()) {
```

Comment:
Either change the signature of `Edge` constructor, or change this to a const reference

Reply:
Thanks for the catch!